app/src/main/java/io/left/meshim/models/Message.java

```diff
@@ -56,6 +56,28 @@
     @ColumnInfo(name = "IsDelivered")
     private boolean isDelivered;
 
+    //file stuff
```

Comment:
Have anything more expressive than "file stuff"? Optionally we can just remove this comment, the naming makes it pretty clear.